flink-python/pyflink/datastream/window.py

```diff
@@ -197,6 +200,31 @@ def __repr__(self):
         return "CountWindow(id={})".format(self.id)
 
 
+class GlobalWindow(Window):
```

Comment:
Need handle properly for GlobalWindow in https://github.com/apache/flink/blob/master/flink-python/pyflink/datastream/data_stream.py#L2135